set.mm

```diff
@@ -255599,11 +255619,15 @@ nonzero elements form a group under multiplication (from which it
 
   ${
     $d s w $.
-    $( A sub-division-ring is a subset of a division ring's set which is a
-       division ring under the induced operation.  If the overring is
-       commutative this is a field; no special consideration is made of the
-       fields in the center of a skew field.  (Contributed by Stefan O'Rear,
-       3-Oct-2015.) $)
+    $( Define the function associating with a ring the set of its
+       sub-division-rings.  A sub-division-ring of a ring is a subset of its
+       base set which is a division ring when equipped with the induced
+       structure (sum, multiplication, zero, and unit).  If a ring is
+       commutative (resp., a field), then its sub-division-rings are
+       commutative (resp., are fields) ( ~ fldsdrgfld ), so we do not make a
+       specific definition for subfields.  (Contributed by Stefan O'Rear,
+       3-Oct-2015.)  TODO: extend this definition to a function with domain
+       ` _V ` or at least ` Ring ` and not only ` DivRing ` . $)
```

Comment:
```suggestion
       ` Ring ` and not only ` DivRing ` . $)
```
I understand your remark, and I know that we do a lot of things like that in set.mm, but I still think that it makes no sense to allow nonsensical things like "the set of sub-division-rings of { 2 } ".

Reply:
(As you want for the present PR.  Non-blocking.)

Reply:
Yes, but you could imagine defining a sub-division-ring of something that's not even a ring. For example, `QQ` is a division ring which is a subset of [the extended real numbers `RR*s`](https://us.metamath.org/mpeuni/df-xrs.html) which is not even a group, but has the necessary structures and operations. 
Isn't this similar to searching sub-division-rings starting with rings which are not themselves division rings?

Anyway, again, in practice, I don't think this will be used on anything else than fields. If you prefer to restrict the domain to something, `Ring` is a good choice because it the most generic structure (but then again why not e.g. semi-rings `SRing`?).

Reply:
> Isn't this similar to searching sub-division-rings starting with rings which are not themselves division rings?

The reason I gave above to use rings instead of division rings as the ambient structure is added simplicity rather than added generality.  Of course, one can always find a more general setting, by doing what is called "centipede mathematics", but if it comes at the cost of simplicity, this may not be worth it.